common/runtime/client_test.go

```diff
@@ -118,6 +118,32 @@ func TestClient_ListRuntimes(t *testing.T) {
 		assert.Equal(t, 4, rp.TotalCount)
 		assert.Len(t, rp.Data, 4)
 	})
+
+	t.Run("Test deprovisioned runtimes limit", func(t *testing.T) {
+		//given
+		params := ListParameters{
```

Comment:
Do we test an execution path where there is no `StateDeprovisioned` in the params, so the limit has no effect (`limitRuntimes` is false).

Reply:
There are two tests without `StateDeprovisioned` in the params: 
- https://github.com/kyma-project/kyma-environment-broker/blob/main/common/runtime/client_test.go#L35
- https://github.com/kyma-project/kyma-environment-broker/blob/main/common/runtime/client_test.go#L97